examples/bcc/bash_readline/bash_readline.go

```diff
@@ -82,19 +83,20 @@ func main() {
 	sig := make(chan os.Signal, 1)
 	signal.Notify(sig, os.Interrupt, os.Kill)
 
-	fmt.Printf("%10s\t%s\n", "PID", "COMMAND")
+	fmt.Printf("%10s\t%s\t%d\n", "PID", "COMMAND", "LOST")
 	go func() {
 		var event readlineEvent
 		for {
-			data := <-channel
-			err := binary.Read(bytes.NewBuffer(data), binary.LittleEndian, &event)
+			receiverData := <-receiverChan
+			lostData := <-lostChan
```

Comment:
This doesn't work as intended: the example does block here until events are lost. Should be a `select` instead.

But same thing as above: could be addressed in a separate pull request.

Reply:
@schu Done. I simply set lost_cb to `nil`. Now it should work.